components/Approval/ApprovalListTable/index.tsx

```
@@ -57,15 +57,16 @@ const ApproverListTableComponent = ({
   const { isMobile, isTablet } = useIsMobile();
   return (
     <div>
-      <div className="mt-2 w-[900px]">
+      <div className='mt-2 w-full'>
```

ESLint

[error] Replace `'mt-2·w-full'` with `"mt-2·w-full"`

(prettier/prettier)